src/themes/baseTheme.ts

```diff
@@ -33,6 +33,12 @@ export const baseTheme = createTheme({
       xxl: { value: '1.8rem' },
       xxxl: { value: '2.0rem' }
     },
+    fonts: {
+      default: {
```

Comment:
Seems like this needed both variable and static set even though we're really just using a static font 🤔